src/nav.config.ts

```diff
@@ -2632,17 +2632,26 @@ const fullNav: FullNav = {
           href: '/reference/go/secrets/secret-put',
         },
         {
-          title: 'Secret.Version()',
-          href: '/reference/go/secrets/secret-version',
-        },
-        {
-          title: 'Secret.Latest()',
-          href: '/reference/go/secrets/secret-latest',
+          title: 'Secret.AccessVersion()',
+          href: '/reference/go/secrets/secret-access-version',
         },
         {
-          title: 'Secret.Version.Access()',
-          href: '/reference/go/secrets/secret-version-access',
+          title: 'Secret.Access()',
+          href: '/reference/go/secrets/secret-access',
         },
+        // TODO: Add redirects for these
```

Comment:
These will need to be added in this PR for tests to pass. Just add them to next config